<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskEditor</class>
 <widget class="QMainWindow" name="MaskEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaskEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="GraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionAdd"/>
   <addaction name="actionSubtract"/>
   <addaction name="actionIntersect"/>
   <addaction name="actionClear"/>
   <addaction name="actionReverse"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEllipse"/>
   <addaction name="actionRect"/>
   <addaction name="actionPolygon"/>
  </widget>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/mask_ellipse.png</normaloff>:/images/mask_ellipse.png</iconset>
   </property>
   <property name="text">
    <string>圆形工具</string>
   </property>
  </action>
  <action name="actionRect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/mask_rectangle.png</normaloff>:/images/mask_rectangle.png</iconset>
   </property>
   <property name="text">
    <string>矩形工具</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/mask_new.png</normaloff>:/images/mask_new.png</iconset>
   </property>
   <property name="text">
    <string>新选区</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/mask_add.png</normaloff>:/images/mask_add.png</iconset>
   </property>
   <property name="text">
    <string>添加到选区</string>
   </property>
  </action>
  <action name="actionSubtract">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/mask_subtract.png</normaloff>:/images/mask_subtract.png</iconset>
   </property>
   <property name="text">
    <string>从选区减去</string>
   </property>
  </action>
  <action name="actionIntersect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/mask_intersect.png</normaloff>:/images/mask_intersect.png</iconset>
   </property>
   <property name="text">
    <string>与选区交叉</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </action>
  <action name="actionReverse">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/mask_reverse.png</normaloff>:/images/mask_reverse.png</iconset>
   </property>
   <property name="text">
    <string>反转</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤消</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>重作</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/polygon.png</normaloff>:/images/polygon.png</iconset>
   </property>
   <property name="text">
    <string>多边形工具</string>
   </property>
   <property name="toolTip">
    <string>多边形工具</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
